Demo/AuthenticatorDemo/ViewController+WordPressAuthenticator.swift

```diff
@@ -76,5 +78,24 @@ extension ViewController {
                 navTitleTextColor: .white
             )
         )
+
+        WordPressAuthenticator.shared.delegate = self
+    }
+
+    // TODO: Need to handle new user flow
+    func newGoogleSignInFlow() {
+        Task.init {
```

Comment:
I might just be spoiled by only working on SPM stuff, but could we just do:
```suggestion
        Task {
```

Reply:
For sure.

2704cd1c